org.eclipse.tm4e.languageconfiguration/src/main/java/org/eclipse/tm4e/languageconfiguration/internal/utils/TextUtils.java

```diff
@@ -1,5 +1,5 @@
 /**
- * Copyright (c) 2015-2017 Angelo ZERR.
+ * Copyright (c) 2015-2022 Angelo ZERR.
```

Comment:
shouldn't this actually say "and others."?

Reply:
I have nothing against adding "and others" text, but IMHO it's something that author is to decide. If @angelozerr wouldn't mind - I can change.